packages/components/src/MultiSelect/subcomponents/MultiSelectToggle/MultiSelectToggle.tsx

```diff
@@ -40,14 +44,17 @@ export const MultiSelectToggle = forwardRef<
        */}
       {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
       <div
-        className={classnames(styles.multiSelectToggle, classNameOverride)}
+        className={classnames(styles.multiSelectToggle, classNameOverride, [
+          status && styles[status],
+        ])}
```

Comment:
The array isn't needed :)
```suggestion
        className={classnames(styles.multiSelectToggle, classNameOverride,
          status && styles[status],
        )}
```